crypto/conf/conf_def.c

```diff
@@ -429,6 +452,11 @@ static int def_load_bio(CONF *conf, BIO *in, long *line)
                 if (!str_copy(conf, psection, &include, p))
                     goto err;
 
+                if (conf->flag_abspath && !ossl_is_absolute_path(include)) {
+                    ERR_raise(ERR_LIB_CONF, CONF_R_RELATIVE_PATH);
+                    goto err;
+                }
+
```

Comment:
I'd recommend considering semantics that enforces the absolute path check only after prepending the include_dir. IMO it would be much more useful.